tests/unit/test_synthDriverHandler.py

```diff
@@ -56,3 +70,12 @@ def test_setSynth_fallbackMode(self):
 			synthDriverHandler.setSynth(synthName, isFallback=True)
 			self.assertEqual(synthName, synthDriverHandler.getSynth().name)
 			self.assertEqual(self._oldSynthConfig, config.conf["speech"]["synth"])
+
+	def test_setSynth_oneCoreSupportsDefaultLangauge(self):

```

Comment:
Should this be two tests? the first part would be clearer if it explicitly showed that the language is set to the supported 'foo' language.